erpnext/hr/doctype/regular_work_summary_group/regular_work_summary_group.json

```diff
@@ -0,0 +1,175 @@
+  "users",
+  "section_break_holidays",
+  "send_emails_frequency",
+  "holiday_list",
+  "column_request",
+  "send_request_emails_at",
+  "send_request_emails_week_day",
+  "send_request_emails_month_day",
+  "send_request_emails_month",
+  "column_summary",
+  "send_summary_emails_at",
+  "send_summary_emails_week_day",
+  "send_summary_emails_month_day",
+  "send_summary_emails_month",
+  "section_email_content",
+  "subject",
+  "message"
+ ],
+ "fields": [
+  {
+   "default": "1",
+   "fieldname": "enabled",
+   "fieldtype": "Check",
+   "label": "Enabled"
+  },
+  {
+   "fieldname": "select_users",
+   "fieldtype": "Section Break",
+   "label": "Select Users"
+  },
+  {
+   "fieldname": "users",
+   "fieldtype": "Table",
+   "label": "Users",
+   "options": "Regular Work Summary Group User",
+   "reqd": 1
+  },
+  {
+   "fieldname": "section_break_holidays",
+   "fieldtype": "Section Break"
+  },
+  {
+   "fieldname": "holiday_list",
+   "fieldtype": "Link",
+   "label": "Holiday List",
+   "options": "Holiday List"
+  },
+  {
+   "default": "What did you work on today?",
+   "fieldname": "subject",
+   "fieldtype": "Data",
+   "label": "Subject"
+  },
+  {
+   "default": "<p>Please share what did you do today. If you reply by midnight, your response will be recorded!</p>",
+   "fieldname": "message",
+   "fieldtype": "Text Editor",
+   "label": "Message"
+  },
+  {
+   "default": "Daily",
+   "fieldname": "send_emails_frequency",
+   "fieldtype": "Select",
+   "in_list_view": 1,
+   "label": "Send Reminder",
+   "options": "Daily\nWeekly\nMonthly\nYearly",
+   "reqd": 1
+  },
+  {
+   "fieldname": "send_request_emails_at",
+   "fieldtype": "Select",
+   "label": "Time",
+   "options": "00:00\n01:00\n02:00\n03:00\n04:00\n05:00\n06:00\n07:00\n08:00\n09:00\n10:00\n11:00\n12:00\n13:00\n14:00\n15:00\n16:00\n17:00\n18:00\n19:00\n20:00\n21:00\n22:00\n23:00"
+  },
+  {
+   "default": "Friday",
+   "depends_on": "eval:doc.send_emails_frequency == \"Weekly\";",
+   "fieldname": "send_request_emails_week_day",
+   "fieldtype": "Select",
+   "label": "Day",
+   "options": "Sunday\nMonday\nTuesday\nWednesday\nThursday\nFriday\nSaturday"
+  },
+  {
+   "default": "1",
+   "depends_on": "eval:in_list([\"Monthly\", \"Yearly\"], doc.send_emails_frequency )",
+   "fieldname": "send_request_emails_month_day",
+   "fieldtype": "Select",
+   "label": "Day",
+   "options": "1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n19\n20\n21\n22\n23\n24\n25\n26\n27\n28\n29\n30\n31\n-1\n-2\n-3\n-4\n-5\n-6\n-7\n-8\n-9\n-10\n-11\n-12\n-13\n-14\n-15"
+  },
+  {
+   "default": "January",
+   "depends_on": "eval:doc.send_emails_frequency == \"Yearly\"",
+   "fieldname": "send_request_emails_month",
+   "fieldtype": "Select",
+   "label": "Month",
+   "options": "January\nFebruary\nMarch\nApril\nMay\nJune\nJuly\nAugust\nSeptember\nOctober\nNovember\nDecember"
+  },
+  {
+   "fieldname": "send_summary_emails_at",
+   "fieldtype": "Select",
+   "label": "Time",
```

Comment:
Why not use Time DocField rather than Select DocField ?

Reply:
That was in Daily Work Summary at least in master branch. If I understand correctly the reason not to use Time for the field is that the scheduler has an hour frequency and consequently there is no practical reason in setting time with second precision.

Reply:
The time is too detailed with second being the smallest unit. The send email at field is meant for scheduler to call a trigger function. This scheduler calls the trigger function each hour. There is no reason to overwhelm user with unnecessary time precision.

Reply:
We have a scheduler activity that triggers every 4 mins.

Reply:
Still doesn't make much sense to have Time field with second precision and backend frequency of 4 min., that would be confusing for the end user.